Content.Shared/Stacks/SharedStackSystem.cs

```diff
@@ -347,6 +311,139 @@ public bool TryAdd(EntityUid insertEnt, EntityUid targetEnt, int count, StackCom
             return true;
         }
 
+        #region Spawning
+
+        /// <summary>
+        ///     Try to split this stack into two. Returns a non-null <see cref="Robust.Shared.GameObjects.EntityUid"/> if successful.
+        /// </summary>
+        public EntityUid? Split(EntityUid uid, int amount, EntityCoordinates spawnPosition, StackComponent? stack = null)
+        {
+            if (!Resolve(uid, ref stack))
+                return null;
+
+            // Try to remove the amount of things we want to split from the original stack...
+            if (!Use(uid, amount, stack))
+                return null;
+
+            // Get a prototype ID to spawn the new entity. Null is also valid, although it should rarely be picked...
+            var prototype = _prototype.TryIndex<StackPrototype>(stack.StackTypeId, out var stackType)
+                ? stackType.Spawn.ToString()
+                : Prototype(uid)?.ID;
+
+            // Set the output parameter in the event instance to the newly split stack.
+            var entity = PredictedSpawnAtPosition(prototype, spawnPosition);
```

Comment:
This won't work because client can't predict interactions with spawned entities atm.